src/cmd-run

```diff
@@ -86,6 +86,23 @@ if [ -z "${VM_DISK}" ]; then
     fi
 fi
 
+# make sure disk path is absolute
+VM_DISK=$(realpath "${VM_DISK}")
+
+if [ -z "${VM_PERSIST_IMG}" ]; then
+    f=$(mktemp)
```

Comment:
one thing that does worry me here is where this is being stored now. It is in the `/tmp/` in the container whereas before it was being handled by `-snapshot` arg to qemu. Where did the snapshot arg put its storage? Is it different? Do we foresee hitting any disk issues?

Reply:
Pushed a new commit which tries `TMPDIR` first then falls back to `/var/tmp`, which matches what `-snapshot` does!